WordPressAuthenticator.podspec

```diff
@@ -41,8 +41,6 @@ Pod::Spec.new do |s|
   s.dependency 'GoogleSignIn', '~> 5.0.2'
   s.dependency 'WordPressUI', '~> 1.7.0'
 
-  # Use a loose restriction that allows both production and beta versions, up to the next major version.
-  # If you want to update which of these is used, specify it in the host app.
-  s.dependency 'WordPressKit', '~> 4.0-beta.0' # Don't change this until we hit 5.0 in WPKit
-  s.dependency 'WordPressShared', '~> 1.9-beta' # Don't change this until we hit 2.0 in WPShared
+  s.dependency 'WordPressKit', '~> 4.14-beta'
+  s.dependency 'WordPressShared', '~> 1.10-beta'
```

Comment:
same here. If you're having trouble getting access to WordPressKit 4.14.-beta.N, use the command `bundle exec pod update WordPressKit` while in the repo (no `rake dependencies` support in Authenticator, sadly).

Reply:
Rolled back!